src/main/webapp/WEB-INF/tags/instructor/results/filterPanel.tag

```diff
@@ -61,6 +61,17 @@
                           </label>
                       </div>
                   </div>
+                  <div data-toggle="tooltip" title="Indicate missing responses">
+                      <div class="checkbox padding-top-0 min-height-0">
+                          <input type="checkbox" 
+                                onchange="this.form.submit()" 
+                                id="indicate-missing-responses-checkbox" 
+                                value="true" 
+                                name="<%=Const.ParamsNames.FEEDBACK_RESULTS_INDICATE_MISSING_RESPONSES%>"
+                                <c:if test="${filterPanel.missingResponsesShown}"> checked</c:if>> 
+                                Indicate Missing Responses
```

Comment:
"Indicate Missing Responses" looks like it's an attribute or a child element of the `input` tag while it in fact is on the same level. This should be indented 4 spaces with respect to the parent `div` element.